### PR TITLE
Add package managers

### PR DESCRIPTION
**Pull request by stibinator**
Added info for Chocolatey users on Windows. Should probably add Homebrew for mac and the major linux package managers, as these are a much simpler way of getting the binaries.

**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description
Added info about how to use the Chocolatey package manager to install the binaries.

### Motivation and Context
Because the instructions currently don't tell users how to easily download and install the drivers I was about to give up in disgust, when I thought to try chocolatey, and it was ridiculously easy. Using a package manager is by far the easiest way to install *and maintain* packages like this. Homebrew and dpkg / rpm /yum etc should be added too. That is if you want to reduce the friction for new users.

### Types of changes

- [* ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ *] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ *] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/webdriver/driver_requirements.en.md [hunk 1]
```diff
@@ -632,4 +632,25 @@ so whilst the web will continue to change, PhantomJS will not be updated.
 This was after Google announced the ability to run Chrome headlessly, 
 something also now offered by Mozilla's Firefox.
 
+### Chocolatey
```
Comment (diemol) [inline docs_source_files/content/webdriver/driver_requirements.en.md:635]: This section should be called in a way that reflects the content of simplifying the installation of drivers through package managers. It is important to add to the description of the section that the Selenium project does not maintain those packages, so no issues should be raised against this repo.

File: docs_source_files/content/webdriver/driver_requirements.en.md [hunk 2]
```diff
@@ -632,4 +632,25 @@ so whilst the web will continue to change, PhantomJS will not be updated.
...
 
+On Windows systems the driver binaries can be installed individually, or as a collection using [chocolatey](https://chocolatey.org/).
```
Comment (diemol) [inline docs_source_files/content/webdriver/driver_requirements.en.md:637]: The PR should also include the package managers for `macOS` and `Linux`

File: docs_source_files/content/webdriver/driver_requirements.en.md [hunk 3]
```diff
@@ -632,4 +632,25 @@ so whilst the web will continue to change, PhantomJS will not be updated.
...
+
+* To find the available drivers, use choco list:
```
Comment (diemol) [inline docs_source_files/content/webdriver/driver_requirements.en.md:639]: Let's avoid adding the commands here, we should link to their official page that shows how to use the packages, if we duplicate things here, they can get easily out of date.

**Review by diemol — CHANGES_REQUESTED** [review]
Thank you for this PR, @stibinator!
Overall, I think it is a good idea to add this, but please have a look at the comments I left.

**Comment by diemol** [comment]
@stibinator, please let us know if you'd like to continue with this PR.

File: docs_source_files/content/webdriver/driver_requirements.en.md (hunk 2 above)
Comment (stibinator) [reply docs_source_files/content/webdriver/driver_requirements.en.md:637]: I'd like to add this, but I don't have access to mac or linux systems to check it on.

Comment (diemol) [reply docs_source_files/content/webdriver/driver_requirements.en.md:637]: To move forward with this, maybe you can leave the placeholders for the sections.

**Comment by diemol** [comment]
This PR has gone stale, we will revisit when comments are addressed.

**Pull request closed without merging**